packages/components/pie-textarea/README.md

```diff
@@ -29,70 +20,20 @@ This component can be easily integrated into various frontend frameworks and cus
 To install `pie-textarea` in your application, run the following on your command line:
 
 ```bash
-# npm
-$ npm i @justeattakeaway/pie-textarea
+npm i @justeattakeaway/pie-textarea
 
-# yarn
-$ yarn add @justeattakeaway/pie-textarea
+yarn add @justeattakeaway/pie-textarea
```

Comment:
Same

Reply:
I think all of the component READMEs point to their own individual packages, so I don't think it needs to be updated as part of this PR